server/autolinkclient/client.go

```diff
@@ -64,3 +68,66 @@ func (c *Client) Add(links ...autolink.Autolink) error {
 
 	return nil
 }
+
+func (c *Client) Delete(links ...string) error {
+	for _, link := range links {
+		queryParams := url.Values{
+			AutolinkNameQueryParam: {link},
+		}
+
+		req, err := http.NewRequest(http.MethodDelete, "/"+autolinkPluginID+"/api/v1/link", nil)
+		if err != nil {
+			return err
+		}
+
+		req.URL.RawQuery = queryParams.Encode()
+
+		resp, err := c.Do(req)
+		if err != nil {
+			return err
+		}
+		defer resp.Body.Close()
+
+		if resp.StatusCode != http.StatusOK {
+			respBody, _ := ioutil.ReadAll(resp.Body)
+			return fmt.Errorf("unable to install autolink. Error: %v, %v", resp.StatusCode, string(respBody))
+		}
+	}
+
+	return nil
+}
+
+func (c *Client) Get(autolinkName string) ([]autolink.Autolink, error) {
+	queryParams := url.Values{
+		AutolinkNameQueryParam: {autolinkName},
+	}
+
+	req, err := http.NewRequest(http.MethodGet, "/"+autolinkPluginID+"/api/v1/link", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	req.URL.RawQuery = queryParams.Encode()
+
+	resp, err := c.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	respBody, err := ioutil.ReadAll(resp.Body)
```

Comment:
can we common out some parts of this function as similar is present in above as well